Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/WANDPowderReduction.py

```diff
@@ -117,9 +117,28 @@ def PyInit(self):
             doc="The mask from this workspace will be applied before reduction",
         )
 
-        self.copyProperties("ConvertSpectrumAxis", ["Target", "EFixed"])
+        self.declareProperty(
+            "EFixed",
+            37.02,
```

Comment:
Having a default value for this here seems strange, it is set correctly from wavelength during loading with LoadWAND. If we are doing this then we should remove https://github.com/mantidproject/mantid/blob/f643296c46e4e95bcc4e747612842c7b2e37c0f4/Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/LoadWAND.py#L91 as it will not be used.

Actually, we should remove the [Wavelength](https://github.com/mantidproject/mantid/blob/f643296c46e4e95bcc4e747612842c7b2e37c0f4/Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/LoadWAND.py#L29) from LoadWAND completely as this won't be used and could add to the confusion for the users.

I'm also surprised that they didn't want the ability to set the wavelength instead of energy.

Reply:
This always overrides the wavelength value set by LoadWAND, therefore we should remove the option in LoadWAND.

Reply:
All the usage examples fail because you removed the OutputWorkspace property.

Reply:
Ah, I see.  I will update the usage example now.

As for the removing `wavelength` in `LoadWAND`, I don't think we should do it here since it is not requested by the beamline.

Reply:
> As for the removing `wavelength` in `LoadWAND`, I don't think we should do it here since it is not requested by the beamline.

I'm requesting it! By making EFixed no longer optionally it's removed the functionally of the Wavelength property in LoadWAND.